CHANGELOG.rst

```diff
@@ -5,6 +5,7 @@ Version History
 v5.25.4
 -------
 
+* Scripts Form Config is not showing button icons `https://github.com/lsst-ts/LOVE-frontend/pull/541`_
```

Comment:
You are missing a `>` after the `541`. Please always check the compiled version of the CHANGELOG to ensure things are properly displayed.